db/schema.rb

```diff
@@ -662,13 +662,28 @@
     t.datetime "created_at"
     t.datetime "updated_at"
     t.tsvector "tsv"
+    t.string   "video_url"
   end
 
   add_index "poll_questions", ["author_id"], name: "index_poll_questions_on_author_id", using: :btree
   add_index "poll_questions", ["poll_id"], name: "index_poll_questions_on_poll_id", using: :btree
   add_index "poll_questions", ["proposal_id"], name: "index_poll_questions_on_proposal_id", using: :btree
   add_index "poll_questions", ["tsv"], name: "index_poll_questions_on_tsv", using: :gin
 
+  create_table "poll_recounts", force: :cascade do |t|
```

Comment:
This table, as well as the tolk ones, should not have been added. Old table schemas on your local database may have the blame, just:

```bash
rake db:reset
rake db:dev_seed
rake db:migrate
```
And you'll see how it goes back to normal (maybe the tipical change for `t.datetime "password_changed_at"` default.. but that must be ignored until we fix it :D )